code/game/objects/items/weapons/twohanded.dm

```diff
@@ -165,7 +165,7 @@
 	sharpness = IS_SHARP
 
 /obj/item/weapon/twohanded/fireaxe/update_icon()  //Currently only here to fuck with the on-mob icons.
-	icon_state = "fireaxe[wielded]"
+	icon_state = "fireax[wielded]"

```

Comment:
Is this seriously an american spelling convention.
Also this will break the icon.